lib/broccoli/app-config.js

```diff
@@ -1,7 +1,3 @@
-/* jshint ignore:start */
```

Comment:
Oh, I looked for the ignore list in the wrong file. [Hello ignore list!](https://github.com/ember-cli/ember-cli/blob/master/.eslintignore)